src/InputWithLabel.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+const InputWithLabel = ({
+    id,
+    children,
+    value,
+    type = 'type',
```

Comment:
I'm not sure about the type and name props but I believe they're not needed here.